WEB_SERVICE/WEB/_INFO/PATH_ENUMERATION/description.md

```diff
@@ -0,0 +1 @@
+This entry provides a clear overview of the structure and how different types in the GraphQL schema are related and can be traversed.
```

Comment:
Too small.

Reply:
This is an info entry 
listing all the query types and their possible paths.
This does not report a vulnerability.
@nmasdoufi-ol could you specify what do you think shoudl be added?